model/model.go

```diff
@@ -18,9 +18,9 @@ import (
 const ModelVersion int = 2
 
 type Model struct {
-	Version  int                    `json:"version"`
-	Objects  map[ObjectName]*Object `json:"types"`
-	Metadata *Metadata              `json:"metadata"`
+	Version  int                                `json:"version"`
+	Objects  map[types.ObjectName]*types.Object `json:"types"`
```

Comment:
Instead of adding the `type.` prefix everywhere, I'd declare aliases in this package:
```go
type ObjectName = stats.ObjectName
type RelationName = stats.RelationName
...
```